installer/Migration/V4_9_0/Migration.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace OrangeHRM\Installer\Migration\V4_9_0;
+
+use Doctrine\DBAL\Types\Types;
+use OrangeHRM\Installer\Util\V1\AbstractMigration;
+
+class Migration extends AbstractMigration
+{
+    /**
+     * @inheritDoc
+     */
+    public function up(): void
+    {
+        if (!$this->getSchemaHelper()->tableExists(['ohrm_theme'])) {
+            $this->getSchemaHelper()->createTable('ohrm_theme')
+                ->addColumn('theme_id', Types::INTEGER, ['Length' => 11, 'Autoincrement' => true])
+                ->addColumn('theme_name', Types::STRING, ['Length' => 100, 'Notnull' => false])
+                ->addColumn('main_logo', Types::BLOB, ['Notnull' => false])
+                ->addColumn('variables', Types::TEXT, ['Notnull' => false])
+                ->setPrimaryKey(['theme_id'])
+                ->create();
+        }
+        $this->getDataGroupHelper()->insertScreenPermissions(__DIR__ . '/permission/screen.yaml');
+        $this->insertMenuItems('Corporate Branding', 'Add Theme', 1, 2, 700, '', 1);
+        $this->insertTheme(1, 'default', '{"primaryColor":"#f28b38","secondaryColor":"#f3f3f3","buttonSuccessColor":"#56ac40","buttonCancelColor":"#848484"}');
+        $this->getSchemaHelper()->addColumn('ohrm_theme', 'social_media_icons', Types::TEXT, [
+            'Notnull' => true,
+            'Default' => 'inline',
+        ]);
+        $this->getSchemaHelper()->addColumn('ohrm_theme', 'login_banner', Types::BLOB);
+
+        $this->createQueryBuilder()
+            ->insert('ohrm_i18n_group')
+            ->values(
+                [
+                    'name' => ':name',
+                    'title' => ':title',
+                ]
+            )
+            ->setParameter('name', 'branding')
+            ->setParameter('title', 'Corporate Branding')
+            ->executeQuery();
+
+        $this->createQueryBuilder()
+            ->delete('ohrm_marketplace_addon')
+            ->andWhere('ohrm_marketplace_addon.plugin_name = :pluginName')
+            ->setParameter('pluginName', 'orangehrmCorporateBrandingPlugin')
+            ->executeQuery();
+
+        $this->updateConfig('4.9', 'instance.version');
+        $this->updateConfig('81', 'instance.increment_number');
```

Comment:
will remove this, remove from other migrations also

Reply:
removed aded to getVersion